pkg/api/egressfirewall/NOTE.md

```diff
@@ -0,0 +1,10 @@
+This directory is generated by
```

Comment:
Why can't we use their library directly?

Reply:
https://issues.redhat.com/browse/DPTP-4064 Cause our repo is in a really bad state and I'm fighting against it since Friday.

Reply:
I am against that idea. If you are fighting with the vendoring, which most probably you need to force a specific version, coping-pasting manually the library into the pkg folder is a really bad idea.

Reply:
I agree but it's a temporary workaround that we are going to eliminate as soon as I manage to solve the dependency issue we have (I'm on it already). Bear delayed his work for several weeks and at some point we had to do something.